terraform/main.mk

```diff
@@ -7,7 +7,23 @@ ENV_DIR ?= $(INFRA_DIR)/env/$(ENV)
 OUTPUT_JSON_FILE = $(INFRA_DIR)/env/$(ENV)/output.json
 TERRAFORM_VERSION ?= "0.12.29"
 
-
+AWS_LIMITS_LIST ?= $$(echo "{ \
```

Comment:
@igorkotof Let's add a quick comment or link to a documentation to describe what it does

Reply:
done